packages/kotti-ui/source/kotti-comment/KtComment.vue

```diff
@@ -165,16 +132,34 @@ export default defineComponent<KottiComment.PropsInternal>({
 							userBeingRepliedTo.value.userName,
 					  ].join(' '),
 			),
-			replyButton: computed(() => translations.value.replyButton),
 			userBeingRepliedTo,
 		}
 	},
 })
 </script>
 
 <style lang="scss" scoped>
-.action__reply {
+.kt-comment {
 	display: flex;
-	align-items: center;
+	flex-flow: row;
+
+	+ .kt-comment {
+		padding-top: var(--unit-1);
+		border-top: 1px solid var(--ui-02);
+	}
+
+	&__content {
+		display: flex;
+		flex: 1;
+		flex-direction: column;
+		margin-left: var(--unit-2);
+
+		&__info {
+			display: flex;
+			width: 100%;
+			font-size: 0.7rem;
+			line-height: 1.2rem;
```

Comment:
Shouldn't this use unit variables?
`font-size: calc(var(--unit-3) + var(--unit-h));`
`line-height: var(--unit-6);`

Reply:
for font-sizes, i don't think we use unit- , i think we use those for spacings (margins , paddings) only

there were variables for the font but they are not prefixed with font sadly

`--font-size` , `font-size-sm`, `font-size-lg`

Reply:
the line-height (I've never also seen it used with units) but we can make a variable for our root-line-height (one may already exist)